Feat/write get agent activity (#277)
holochain/wind-tunnel: scenarios/write_get_agent_activity/README.md

## write_get_agent_activity

### Description

An agent creates entries, while another agent gets their agent activity with `get_agent_activity`.

### Suggested command

You can run the scenario locally with the following command:

```bash
RUST_LOG=info MIN_AGENTS=2 cargo run --package write_get_agent_activity -- --agents 2 --behaviour write:1 --behaviour get_agent_activity:1 --duration 60

> ```

Reviewer: Possibly worth explicitly mentioning the interaction between sync and `get_agent_activity` here? If the two nodes reach sync with each other then the writer will publish their created data and `get_agent_activity` should often have most of the data it needs locally. So this is (correctly) more a test of database query performance and the code path through the host function than it is about `get_agent_activity` in general?

Author: Expended the description: https://github.com/holochain/wind-tunnel/pull/277/commits/f37d72813069d02aea4f0a5330288af66f9180b3

It does seem like this scenario is a generalized demonstration of the performance of `get_agent_activity`, through the various conditions it would find in real usage. Not sure if of the under-the-hood implementation details are really important, but I guess its not a bad thing to give any reviewers some hints to explain what they're seeing.

## write_get_agent_activity

### Description

> An agent creates an entry, then gets their own agent activity with `get_agent_activity`

Author: ```suggestion
An agent creates entries, while another agent gets their agent activity with `get_agent_activity`.
```
